src/GrafanaDashboardDeploy/src/deploy-pack.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Is there only 1 giant file?  This might cause some merge conflict mess.  Would it be possible to split it up a bit? Maybe have a file per dashboard/datasource?  Otherwise it's going to be one giant, hard to deal with file.